spec/rubocop/cop/naming/method_name_spec.rb

```diff
@@ -118,9 +179,40 @@ def on_selection_cleared(arg)
     end
   end
 
+  shared_examples 'multiple attr methods' do |enforced_style|
+    it 'registers an offense for camel case methods names in attr.' do
+      expect_offense(<<~RUBY)
+        attr :my_method, :myMethod
+             ^^^^^^^^^^^^^^^^^^^^^ Use #{enforced_style} for method names.
```

Comment:
Shouldn't we highlight just the second symbol here?

Reply:
Will check it. Do you know maybe some cops that have similar behavior? 

So it should be something like that ? 

```
attr :my_method, :myMethod, my_snake_method, : myCamelMethod
	                           ^^^^^^^^^^^^                                   ^^^^^^^^^^^^^^^^^^^ Use #{enforced_style} for method names.
```

Reply:
Yep. See https://github.com/rubocop-hq/rubocop/pull/7385